recipes/quetz-sql-authenticator/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "quetz-sql-authenticator" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/quetz-sql-authenticator-{{ version }}.tar.gz
+  sha256: 49212cf94c664723df9c7d24b1f44e87b06daa6f1522692095f6ceef7e6f8a6d
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win or py<38]
```

Comment:
why don't we build for windows? isn't this a simple `noarch` package?

Reply:
Apparently quetz is not available on Windows, see [this run](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=659110&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf&l=359).